dask/optimize.py

```diff
@@ -34,9 +34,11 @@ def cull(dsk, keys):
     return dict((k, v) for k, v in dsk.items() if k in seen)
 
 
-def fuse(dsk):
+def fuse(dsk, keys=None):
```

Comment:
I don't have a strong opinion on this.  One can view this as keys to *keep* in the dask, or keys to ignore when performing the fuse operation.  I slightly favor `keys`, but am happy to change it to something more descriptive.